DEVELOPING.md

```diff
@@ -0,0 +1,44 @@
+## Developing
+
+With the new inclusion of Gradle replacing ant (bye bye 2014) this brings some more modern features. Running any of the commands will automatically install the Gradle wrapper, no other install needed. This uses Gradle 6.5.1 and Java 11.
+
+### Run The Demo
+
+```
+./gradlew run
```

Comment:
I tried this and got a "Permission denied" error.
After:    chmod +x ./gradlew
then it worked.